lib/osf-components/addon/components/contributors/card/editable/component.ts

```diff
@@ -15,4 +17,10 @@ export default class ContributorsCardEditable extends Component {
 
     // private properties
     permissionOptions = Object.values(Permission);
+    @tracked showDropdown = false;
+
+    @action
+    toggleDropdown() {
+        this.toggleProperty('showDropdown');
```

Comment:
with `showDropdown` being `tracked`, you might want to reassign as such
```suggestion
        this.showDropdown = !this.showDropdown;
```
`toggleProperty` uses  legacy `@ember/component` `set()` underneath